operations/kubernetes/Kyverno.md

```diff
@@ -456,3 +456,20 @@ and [match/exclude](https://release-1-10-0.kyverno.io/docs/writing-policies/matc
 
     If the issue still persists after restarting the Kyverno pods, then HPE recommends disabling the background scanning in the Kyverno policy.
     Set `background` to `false` to disable background scanning in the Kyverno policy.
+* Multiple Kyverno admission reports may cause etcd downtime during upgrade to CSM 1.6.
```

Comment:
This should describe the symptoms an admin would observe during an upgrade that would indicate this problem is happening. 

Specifically, it should explain:
1. How the etcd downtime will manifest itself during the upgrade
2. If there is a way to determine that the etcd downtime is caused by this versus something else